src/readers/morphologySWC.cpp

```diff
@@ -206,6 +206,14 @@ class SWCBuilder
         const Sample& child1 = _children[0];
         const Sample& child2 = _children[1];
 
+        bool isSuited = std::fabs(child1.diameter - d) < morphio::epsilon &&
+                        std::fabs(child2.diameter - d) < morphio::epsilon &&
+                        std::fabs(child1.point[0] - x) < morphio::epsilon &&
+                        std::fabs(child2.point[0] - x) < morphio::epsilon &&
+                        std::fabs(child1.point[2] - z) < morphio::epsilon &&
+                        std::fabs(child2.point[2] - z) < morphio::epsilon;
+        if (!isSuited)
```

Comment:
Same you should explain why you don t throw

Reply:
ok